index.js

```diff
@@ -15,5 +15,11 @@ exports.onHandleCode = function onHandleCode(ev) {
     .replace(/exports\s*\.\s*([_\d\w]+)\s*=\s*\1\s*;/g,
              'export { $1 };')
     .replace(/exports\s*\.\s*([_\d\w]+)\s*=/g,
-             'export let $1 =');
+             'export let $1 =')
+    .replace(/^(?:const|var|let)\s(\w+)\s*=\s*require\s*\(\s*(.*?)\s*\)/gm,
+             'import $1 from $2')
+    .replace(/^(?:const|var|let)\s*(\{(?:[\s\S]*?)\})\s*=\s*require\s*\(\s*(.*?)\s*\)/gm,
+             'import $1 from $2')
+    .replace(/^require\s*\(\s*(.*?)\s*\)/gm,
```

Comment:
Side note, this may produce invalid ES6 code when require is anywhere but the top of a file (like inside a function). Not sure how ESDoc would handle that, but it's worth adding a test for it to see what happens.

Reply:
That's on purpose. Check that the expression is for a `require` that doesn't return a value. It would be a pretty edge case you had a `require` without return on the middle of the code.